src/wallet/api/wallet.cpp

```diff
@@ -1518,14 +1518,26 @@ std::string WalletImpl::getMultisigKeyExchangeBooster(const std::vector<std::str
     return string();
 }
 
-bool WalletImpl::exportMultisigImages(string& images) {
+bool WalletImpl::exportMultisigImages(string& images, std::string filename, bool ascii) {
     try {
         clearStatus();
         checkMultisigWalletReady(m_wallet);
 
         auto blob = m_wallet->export_multisig();
-        images = epee::string_tools::buff_to_hex_nodelimer(blob);
-        return true;
+        if (filename.empty()) {
+            if (ascii) {
```

Comment:
This seems like quite a different use case than the existing function, I think a separate function would be better (also suggest `PEM` as the qualifier, rather than `ascii`, as all the formats are ASCII data).